sporkfish/searcher/minimax.py

```diff
@@ -101,10 +116,13 @@ def _build_move_order_heuristic(
             return MvvLvaHeuristic(board)
         elif order_type is MoveOrderMode.KILLER_MOVE:
             return KillerMoveHeuristic(board, self._killer_moves, depth)  # type: ignore
+        elif order_type is MoveOrderMode.HISTORY:
+            return HistoryHeuristic(board, self._history_table)
         elif order_type is MoveOrderMode.COMPOSITE:
             return CompositeHeuristic(
                 board,
                 self._killer_moves,  # type: ignore
+                self._history_table,
```

Comment:
Suspect this needs `# type: ignore` too.

Reply:
adding this gives the error code:
`sporkfish/searcher/minimax.py:118: error: Unused "type: ignore" comment `

Reply:
I see - my bad - in which case pls ignore as mypy seems happy